docs/examples.md

```diff
@@ -0,0 +1,145 @@
+# Examples
+
+For consistency and clarity of the following examples, we're going to use a simplified dbt project. In practice, the model governance features describe are _most_ beneficial for large dbt projects that are struggling to scale. 
+
+We will give a basic example for each command, but to see the full list of additional flags you can add to a given command, check out the [commands page](commands.md). 
+
+!!! note
+    One helpful flag that you can add to all of the commands is `--read-catalog`, which will skip the `dbt docs generate` step and instead read the local `catalog.json` file - this will speed up the time it takes to run the `dbt-meshify` commands but relies on your local `catalog.json` file being up-to-date. 
```

Comment:
you can also set the `DBT_MESHIFY_READ_CATALOG` env var, which would be good to add here!